kodilla-hibernate/src/test/java/com/kodilla/hibernate/manytomany/dao/CompanyDaoTestSuite.java

```diff
@@ -39,25 +39,25 @@ void testSaveManyToMany() {
 
         //When
         companyDao.save(softwareMachine);
-        long softwareMachineId = softwareMachine.getId();
+        int softwareMachineId = softwareMachine.getId();
         companyDao.save(dataMaesters);
-        long dataMaestersId = dataMaesters.getId();
+        int dataMaestersId = dataMaesters.getId();
         companyDao.save(greyMatter);
-        long greyMatterId = greyMatter.getId();
+        int greyMatterId = greyMatter.getId();
 
         //Then
         assertNotEquals(0, softwareMachineId);
         assertNotEquals(0, dataMaestersId);
         assertNotEquals(0, greyMatterId);
 
         //CleanUp
-        //try {
-        //    companyDao.deleteById(softwareMachineId);
-        //    companyDao.deleteById(dataMaestersId);
-        //    companyDao.deleteById(greyMatterId);
-        //} catch (Exception e) {
-        //    //do nothing
-        //}
+        try {
```

Comment:
to nie jest część zadania 17.3, więc raczej powinniśmy tutaj przywrócić wersję z gałęzi głównej

Reply:
Solved!